src/incubator/TextField/index.tsx

```diff
@@ -200,7 +200,7 @@ const TextField = (props: InternalTextFieldProps) => {
               retainSpace
             />
           )}
-          {showCharCounter && <CharCounter maxLength={others.maxLength} charCounterStyle={charCounterStyle}/>}
+          {showCharCounter && <CharCounter maxLength={others.maxLength} charCounterStyle={charCounterStyle} />}
```

Comment:
please remove space, it cause a lint error 
```suggestion
          {showCharCounter && <CharCounter maxLength={others.maxLength} charCounterStyle={charCounterStyle}/>}
```

Reply:
Womp womp 😞 

Fixed! ✅